pkg/storage/storage.go

```diff
@@ -75,9 +76,9 @@ type RelationshipTupleReader interface {
 	// There is NO guarantee on the order returned on the iterator.
 	Read(ctx context.Context, store string, tupleKey *openfgav1.TupleKey) (TupleIterator, error)
 
-	// ReadPage functions similarly to Read but includes support for pagination. It takes additional
-	// pagination parameters and returns a slice of tuples along with a continuation token, which may
-	// not be empty. This token can be used for retrieving subsequent pages of data.
+	// ReadPage functions similarly to Read but includes support for pagination. It takes
+	// mandatory pagination options (pageSize can be zero :/)
```

Comment:
I just noticed this in this PR that the interface of ReadPage is really weird, if you do `ReadPage(.... &storageOptions{From: "cont-token"})` then the storage gets `pageSize = 0`. I think implementations should handle this, in a different PR

Reply:
@adriantam